docs/source/index.rst

```diff
@@ -1,30 +1,78 @@
-Dask - A Task Scheduling Specification
-======================================
+Dask - Parallel Processing Through Blocked Algorithms
+=====================================================
 
-Dask encodes directed acyclic graphs of task-oriented computations
-using ordinary Python dicts, tuples, and functions.
+Dask enables parallel processing on larger-than-memory datasets.
+Dask collections mimic NumPy, Pandas, and Toolz interfaces but operate smoothly
+from disk and use all of your cores.  Dask primarily operates on a single
+machine and is trivial to set up::
```

Comment:
"Dask primarily operates on a single machine" might turn people off who want `dask.distributed`. Maybe clarify that dask can operate on distributed systems but is not reliable yet.